src/cephadm/cephadmlib/daemons/smb.py

```diff
@@ -784,6 +842,10 @@ def _write_ctdb_stub_config(self, path: pathlib.Path) -> None:
             stub_config['ctdb']['log_level'] = self._cfg.ctdb_log_level
         if self._cfg.ctdb_port != Ports.CTDB.value:
             stub_config['ctdb']['ctdb_port'] = self._cfg.ctdb_port
+        if self._cfg.bind_to:
+            stub_config['ctdb']['conf_file_includes'] = [
+                '/etc/ctdb/smb.interfaces.conf'
+            ]
```

Comment:
This results in the following line in _/etc/samba/smb.conf_:
```
include = /etc/ctdb/smb.interfaces.conf
```
This looks a bit odd to see a configuration file (with parameters for _smb.conf_) from _/etc/ctdb_ mentioned to be included within _smb.conf_. I would put such an include file under _/etc/samba_ and not _/etc/ctdb_.

Reply:
FWIW I find the whole section of code that manages the files and mounts to be a big mess. More than once I've started trying to clean it up, usually by trying to define a set of types to manage mounts and dirs only to give up for various reasons. This whole area is collecting technical debt quickly *sigh*